src/Viewer.js

```diff
@@ -333,7 +333,7 @@ export class Viewer {
     setupCamera() {
         if (!this.usingExternalCamera) {
             const renderDimensions = new THREE.Vector2();
-            this.getRenderDimensions(renderDimensions);
```

Comment:
is this refactor necessary? Is that fixing something related to processing files? It doesn't seem to be.